Alicerce/Sources/Logging/Log+NodeProvider.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  Log+NodeProvider.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 07/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+public extension Log {
+
+    public class NodeProvider : LogProvider {
+
+        private static let dispatchQueueLabel = "Alicerce-Log"
+        private static let defaultRequestTimeout: TimeInterval = 0
+
+        private let serverURL: URL
+        private let operationQueue = OperationQueue()
+        private let requestTimeout: TimeInterval
+
+        public var minLevel: Log.Level = .error
+        public var formatter: LogItemFormatter = Log.ItemStringFormatter()
+
+        public var logItemsSent: Int = 0
+
+        //MARK:- lifecycle
+
+        public init(serverURL: URL,
+                    dispatchQueue: DispatchQueue = DispatchQueue(label: NodeProvider.dispatchQueueLabel),
+                    requestTimeout: TimeInterval = NodeProvider.defaultRequestTimeout) {
+            self.serverURL = serverURL
+            self.requestTimeout = requestTimeout
+            self.operationQueue.underlyingQueue = dispatchQueue
+        }
+
+        //MARK:- private methods
+
+        internal func send(payload: Data, completion: @escaping (_ success: Bool) -> Void) {
+
+            let session = URLSession(configuration: URLSessionConfiguration.default,
+                                     delegate: nil, delegateQueue: operationQueue)
+
+            var request = URLRequest(url: serverURL,
```

Comment:
yes, it's just an example, that may be a base for easy on-device-remote-logging, so that we can quickly visualise log messages from devices on a mac that runs a small node script. 
Any other 3rd party logging provider should implement its own ```LogDestination```class with its specifics.